extensions/commands/recipe/README.md

```diff
@@ -36,3 +36,32 @@ updating moltenvk/1.2.0 to moltenvk/1.2.2 in conan-center-index/recipes/qt/6.x.x
 updating libjpeg-turbo/2.1.4 to libjpeg-turbo/2.1.5 in conan-center-index/recipes/qt/6.x.x/conanfile.py:386
 Successfully bumped the dependencies of recipe conan-center-index/recipes/qt/6.x.x/conanfile.py
 ```
+
+#### [Create SBOM](cmd_create_sbom.py)
+
+Creates an SBOM in CycloneDX 1.4 JSON format.
+
+**Parameters**
+* **format** _Required_: `cyclonedx_1.4_json`
+* supports all other arguments used by `conan graph`, see `conan recipe:create-sbom --help`
+
+**Dependencies**
+The command requires the packages `packageurl` and `cyclonedx`.
+You can install them via
+
+```bash
+pip install packageurl-python cyclonedx-python-lib
+```
+
+Usage:
+
+```
```

Comment:
please add code fence language `shellSession`
like so
```text
```shellSession
$ cd conan-center-index/recipes/gmp/all
$ conan recipe:create-sbom -f cyclonedx_1.4_json .
{"components": [{"bom-ref": 
...
````

Reply:
Language is now specified